common/djangoapps/microsite_configuration/admin.py

```diff
@@ -15,4 +15,24 @@ class Meta(object):  # pylint: disable=missing-docstring
         model = Microsite
 
 
+class MicrositeHistoryAdmin(admin.ModelAdmin):
```

Comment:
OK, but if we make Microsites model a config_model, then that change will not be backwards compatible with your production environment. I've been presuming that you are currently running a db-backed environment. Would you be OK with an incompatible change?